Unresolved Imports when Building Docker Image

![image](https://user-images.githubusercontent.com/40243230/111951697-988f4680-8b16-11eb-89fa-fc5df1bc4dff.png)

After running command `docker-compose -f example/docker-compose.yml up`, I get this error.

Is there any solution to this? Thanks

I can confirm this also happened when building the image manually.
`docker build -t tapalogi/tapa-micro-mailer:0.1.0-beta.0 .`

```bash
#18 122.4    Compiling lettre v0.10.0-beta.3
#18 127.9    Compiling tapa-micro-mailer v0.1.0-beta.0 (/app)
#18 128.2 error[E0432]: unresolved imports `lettre::Tokio02Connector`, `lettre::Tokio02Transport`
#18 128.2   --> src/mailer/mod.rs:13:61
#18 128.2    |
#18 128.2 13 | use lettre::{Address, AsyncSmtpTransport, Message as Email, Tokio02Connector, Tokio02Transport};
#18 128.2    |                                                             ^^^^^^^^^^^^^^^^  ^^^^^^^^^^^^^^^^ no `Tokio02Transport` in the root
#18 128.2    |                                                             |
#18 128.2    |                                                             no `Tokio02Connector` in the root
#18 128.2    |                                                             help: a similar name exists in the module: `Tokio02Executor`
#18 128.2
#18 128.6 error: aborting due to previous error
#18 128.6
#18 128.6 For more information about this error, try `rustc --explain E0432`.
#18 128.7 error: could not compile `tapa-micro-mailer`
#18 128.7
#18 128.7 To learn more, run the command again with --verbose.
------
executor failed running [/bin/sh -c RUSTFLAGS="-C link-args=-s" cargo build --release]: exit code: 101
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -5,7 +5,7 @@ authors = ["Aditya Kresna <kresna@tapalogi.com>"]
 edition = "2018"
 
 [dependencies.lettre]
-version = "0.10.0-alpha.5"
+version = "=0.10.0-alpha.5"
 default-features = false
 features = [
     "builder",

